features/editpage.feature

```diff
@@ -0,0 +1,8 @@
+Feature: Post Edit Page
+    Implement an edit page that can be accessed from the detail
+    page if logged in.
+
+    Scenario: Move to the edit page
+        Given a posts entry detail with a title New Title
+        When I click the edit button
+        Then I move to the edit page
```

Comment:
Likewise, the two editing "features" files should really be one "feature" file with several scenarios in it.

In addition to what you have here, it would be good to have tests that demonstrate the fact that an anonymous user cannot edit an existing entry, that providing bad values for title and or text has appropriate consequences, and so on.  Test not only what you think should happen, but what you think might happen.  Expose failure points with tests.